fearless/Modules/Send/SendInteractor.swift

```diff
@@ -217,13 +254,25 @@ extension SendInteractor: AccountInfoSubscriptionAdapterHandler {
 }
 
 extension SendInteractor: PriceLocalStorageSubscriber, PriceLocalSubscriptionHandler {
-    func handlePrice(result: Result<PriceData?, Error>, priceId: AssetModel.PriceId) {
+    func handlePrice(result: Swift.Result<PriceData?, Error>, priceId: AssetModel.PriceId) {
         output?.didReceivePriceData(result: result, for: priceId)
     }
 }
 
 extension SendInteractor: ExtrinsicFeeProxyDelegate {
-    func didReceiveFee(result: Result<RuntimeDispatchInfo, Error>, for _: ExtrinsicFeeId) {
+    func didReceiveFee(result: Swift.Result<RuntimeDispatchInfo, Error>, for _: ExtrinsicFeeId) {
         output?.didReceiveFee(result: result)
     }
 }
+
+extension SendInteractor: TransferFeeEstimationListener {
```

Comment:
perhaps be better update func didReceiveFee(fee: BigUInt) with func didReceiveFee(fee: Result<BigUInt, Error?>)